templates/500.html

```diff
@@ -6,7 +6,7 @@
   <link rel="shortcut icon" href="{% static "img/favicon.ico" %}">
 </head>
 <h2>Internal Server Error</h2>
-Something is wrong. We don't really know what.<br/> If the problem persists please <a href="http://dev.comics.org/bugs/enter_bug.cgi?product=GCD Online&bug_file_loc={{ request.build_absolute_uri }}">let us know.</a>
+Something is wrong. We don't really know what.<br/> If the problem persists please <a href="https://github.com/GrandComicsDatabase/gcd-django/issues/new?body=Occurred+at+{{ request.build_absolute_uri }}">let us know.</a>
```

Comment:
It wasn't there before, but I think you have to encode parameters used in urls, like this:
{{ request.build_absolute_uri|urlencode }}
(and the same in 404.html)